.github/workflows/ci.yml

```diff
@@ -1,12 +1,11 @@
 name: CI
 
 on:
-  workflow_dispatch: {}
   push:
     branches:
       - master
     tags:
-      - '*'
+      - v*
```

Comment:
This can't be a more strict regex right?

Reply:
We can but I'm not sure if it will add anything.

Reply:
I think it'd be good just to make sure this doesn't accidentally get released for some other tag. Better to be overly cautious here.